src/ChatClosedButton.js

```diff
@@ -15,6 +15,6 @@ export default class ChatClosedButton extends React.Component {
 }
 
 ChatClosedButton.propTypes = {
-  scope: PropTypes.oneOfType(["lobby", "round"]).isRequired,
+  scope: PropTypes.oneOfType([s=>{s=="lobby"}, s=>{s=="round"}]).isRequired,
```

Comment:
This looks like a bug from a previous commit, but we might as well fix it. I believe it should be:

```js
  scope: PropTypes.oneOf(["lobby", "round"]).isRequired,
```

Reply:
The `.oneOf(["lobby", "round"])` syntax was giving me warnings, saying that `oneOf` expected a list of functions, not a list of strings, so I changed it to this. Maybe I've got a different version of `prop-types`.